src/fdk.ts

```diff
@@ -48,6 +48,38 @@ async function checkTokenExpired(auth_token) {
     }
 }
 
+const extraSentryDetails = () => {
```

Comment:
@manishprajapati52 We don't require this.

Reply:
In many sentry, we didn't know which command user was executing when sentry triggered. I added this to make our debugging process faster. Do we have that data already in sentry? Should I remove this? @vivek-gofynd